db/migrate/20220531172911_add_index_to_contact_info_value.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToContactInfoValue < ActiveRecord::Migration[5.2]
+  def change
+    add_index :contact_infos, [:value, :type], unique: true
```

Comment:
removed this migration from this PR, so going to resolve this - we'll cross this bride in a future PR (probably the rubocop one.. since that's what started this)